templates/designToConfigParser.jinja

```diff
@@ -103,10 +103,34 @@ bool {{functionPrefix}}configure{{class_name}} (
   mapping addressActiveControl,
 	mapping connectionSettings)
 {
-  DebugTN("Configure.{{class_name}} called");
+  DebugTN("Configure.{{class_name}} called from XML");
   string name;
   xmlGetElementAttribute(docNum, childNode, "name", name);
```

Comment:
there is a check for an error message for this call in the master. Could you unmerge the master to your PR?

Reply:
Rebased to master